wave/src/main/resources/org/waveprotocol/wave/client/wavepanel/view/dom/full/Blip.css

```
@@ -453,22 +453,50 @@
   margin-left: 0;
-  margin-bottom: 0.35em;
-  align-self: flex-start;
+  margin-bottom: 0;
+}
+.compact-inline-blips [data-depth="0"] .metabar,
+.compact-inline-blips [data-depth="1"] .metabar,
+.compact-inline-blips [data-depth="2"] .metabar {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+  margin-left: 0;
```

Stylelint (17.7.0)

[error] Duplicate property "margin-left" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)